REST Bindings/openapi.yaml

```diff
@@ -2654,6 +2723,25 @@ components:
                     enum:
                       - epcisException:SecurityException
               - $ref: '#/components/schemas/RFC7807ProblemResponseBody'
+        application/problem+xml:
```

Comment:
why don't we reuse schema here?

Reply:
same here just a copy for xml content-type referencing same schema